tools/installpsh-gentoo.sh

```diff
@@ -0,0 +1,220 @@
+#!/bin/bash
+
+#call this code direction from the web with:
+#bash <(wget -O - https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/installpsh-gentoo.sh) ARGUMENTS
+#bash <(curl -s https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/installpsh-gentoo.sh) <ARGUMENTS>
+
+#Usage - if you do not have the ability to run scripts directly from the web,
+#        pull all files in this repo folder and execute, this script
+#        automatically prefers local copies of sub-scripts
+
+#Completely automated install requires a root account or sudo with a password requirement
+
+#Switches
+# -skip-sudo-check    - use sudo without verifying its availability (hard to accurately do on some distros)
+# -preview            - installs the latest preview release of PowerShell side-by-side with any existing production releases
+
+#gitrepo paths are overrideable to run from your own fork or branch for testing or private distribution
+
+VERSION="1.2.0"
```

Comment:
It would be nice if you added a test for this, but you need a gentoo container that will work in azure devops.

meaning it needs node, sudo, and adduser (I might be forgetting some, here is what we check for, some are for our tests, https://github.com/PowerShell/PowerShell-Docker/blob/master/tests/container.tests.ps1#L567-L578)

Reply:
I will look into this, I've never worked with azure but I've done work with docker containers etc.. And it looks like some base images for gentoo are available.

Reply:
You would probably have to build it.